pkg/lib/host-connector/vmware_host_connector.go

```diff
@@ -256,51 +257,39 @@ func getEventLogInfo(parsedEventLogEntry types.TpmEvent) types.EventLog {
 
 	log.Trace("vmware_host_connector:getEventLogInfo() Entering")
 	defer log.Trace("vmware_host_connector:getEventLogInfo() Leaving")
-	eventLog := types.EventLog{Value: intArrayToHexString(parsedEventLogEntry.EventDetails.DataHash)}
-	eventLog.Info = make(map[string]string)
+	eventLog := types.EventLog{Measurement: intArrayToHexString(parsedEventLogEntry.EventDetails.DataHash)}
 
 	if parsedEventLogEntry.EventDetails.VibName != nil {
-		eventLog.Label = *parsedEventLogEntry.EventDetails.ComponentName
-		eventLog.Info["EventType"] = TPM_SOFTWARE_COMPONENT_EVENT_TYPE
-		eventLog.Info["ComponentName"] = COMPONENT_PREFIX + *parsedEventLogEntry.EventDetails.ComponentName
-		eventLog.Info["EventName"] = VIM_API_PREFIX + TPM_SOFTWARE_COMPONENT_EVENT_TYPE + DETAILS_SUFFIX
-		if parsedEventLogEntry.EventDetails.VibName != nil {
-			eventLog.Info["PackageName"] = *parsedEventLogEntry.EventDetails.VibName
-		}
-		if parsedEventLogEntry.EventDetails.VibVendor != nil {
-			eventLog.Info["PackageVendor"] = *parsedEventLogEntry.EventDetails.VibVendor
-		}
-		if parsedEventLogEntry.EventDetails.VibVersion != nil {
-			eventLog.Info["PackageVersion"] = *parsedEventLogEntry.EventDetails.VibVersion
-		}
-
-		if parsedEventLogEntry.PcrIndex == 19 {
-			eventLog.Info["FullComponentName"] = "componentName." + (*parsedEventLogEntry.EventDetails.ComponentName)[0:strings.Index(*parsedEventLogEntry.EventDetails.ComponentName, ".")] + "-" +
-				*parsedEventLogEntry.EventDetails.VibName + "-" + *parsedEventLogEntry.EventDetails.VibVersion
+		eventLog.TypeID = VIB_NAME_TYPE_ID
+		eventLog.TypeName = *parsedEventLogEntry.EventDetails.ComponentName
+		eventLog.Tags = append(eventLog.Tags, COMPONENT_PREFIX+*parsedEventLogEntry.EventDetails.ComponentName)
```

Comment:
Not sure of COMPONENT_PREFIX value here, see if separator required in this concat operation?

Reply:
Earlier, in pcr manifest section,we have used component name value with the COMPONENT_PREFIX("componentName.") in info section 
**"info": {
    "ComponentName": "componentName.mtip32xx.v00",
	...}**
so,we have added the same value in tags section.

And, we have used component name without COMPONENT_PREFIX("componentName.") in label
**"label": "mtip32xx.v00"** 
and now, we are using this same value in type_name.